### PR TITLE
Add gcp cloud cost connector

### PR DESCRIPTION
**Pull request by rathodmeetsatish**
## Describe your changes
Add gcp cloud cost connector

## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- Sub Category Field Check: `trigger subcategoryfieldcheck`

### CODE REVIEW

File: docs/data-sources/platform_connector_gcp_cloud_cost.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "harness_platform_connector_gcp_cloud_cost Data Source - terraform-provider-harness"
+subcategory: "Next Gen"
+description: |-
+  Datasource for looking up an GCP Cloud Cost Connector.
```
Comment (rijajoo) [inline docs/data-sources/platform_connector_gcp_cloud_cost.md:6]: a*

File: docs/data-sources/platform_connector_gcp_cloud_cost.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+---
+
+# harness_platform_connector_gcp_cloud_cost (Data Source)
+
+Datasource for looking up an GCP Cloud Cost Connector.
```
Comment (rijajoo) [inline docs/data-sources/platform_connector_gcp_cloud_cost.md:11]: Please review all places it says an GCP

**Review by RajBaviskar — APPROVED** [review]

File: docs/data-sources/platform_connector_gcp_cloud_cost.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+- `identifier` (String) Unique identifier of the resource.
+- `name` (String) Name of the resource.
+- `org_id` (String) Unique identifier of the Organization.
+- `project_id` (String) Unique identifier of the Project.
+
+### Read-Only
+
+- `billing_export_spec` (List of Object) Returns Billing details. (see [below for nested schema](#nestedatt--billing_export_spec))
+- `description` (String) Description of the resource.
+- `features_enabled` (Set of String) Which feature to enable among BILLING, OPTIMIZATION, VISIBILITY
```
Comment (rashminandasahoo) [inline docs/data-sources/platform_connector_gcp_cloud_cost.md:35]: Indicates which features to enable among Billing, Optimization, and Visibility.
Can multiple features be enabled?

File: docs/data-sources/platform_connector_gcp_cloud_cost.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+- `gcp_project_id` (String) GCP Project Id.
+- `id` (String) The ID of this resource.
```
Comment (rashminandasahoo) [inline docs/data-sources/platform_connector_gcp_cloud_cost.md:37]: Id of the resource.

File: docs/data-sources/platform_connector_gcp_cloud_cost.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+- `service_account_email` (String) Service Account Email.
```
Comment (rashminandasahoo) [inline docs/data-sources/platform_connector_gcp_cloud_cost.md:38]: Email corresponding to the Service Account.

File: docs/data-sources/platform_connector_gcp_cloud_cost.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+- `tags` (Set of String) Tags to associate with the resource. Tags should be in the form `name:value`.
```
Comment (rashminandasahoo) [inline docs/data-sources/platform_connector_gcp_cloud_cost.md:39]: Tags to use for the resource. Tags should be in the form `name:value`.

File: docs/resources/platform_connector_gcp_cloud_cost.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `features_enabled` (Set of String) Which feature to enable among BILLING, OPTIMIZATION, VISIBILITY
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_gcp_cloud_cost.md:37]: Refer to the previous comment.

File: docs/resources/platform_connector_gcp_cloud_cost.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+- `gcp_project_id` (String) GCP Project Id.
+- `identifier` (String) Unique identifier of the resource.
+- `name` (String) Name of the resource.
+- `service_account_email` (String) Service Account Email.
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_gcp_cloud_cost.md:41]: Refer to the previous comment.

File: docs/resources/platform_connector_gcp_cloud_cost.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+
+### Optional
+
+- `billing_export_spec` (Block List, Max: 1) Returns Billing details. (see [below for nested schema](#nestedblock--billing_export_spec))
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_gcp_cloud_cost.md:45]: Returns billing details.

File: docs/resources/platform_connector_gcp_cloud_cost.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+- `description` (String) Description of the resource.
+- `org_id` (String) Unique identifier of the Organization.
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_gcp_cloud_cost.md:47]: Unique identifier of the organization.

File: docs/resources/platform_connector_gcp_cloud_cost.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+- `project_id` (String) Unique identifier of the Project.
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_gcp_cloud_cost.md:48]: Unique identifier of the project.

File: docs/resources/platform_connector_gcp_cloud_cost.md [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+- `tags` (Set of String) Tags to associate with the resource. Tags should be in the form `name:value`.
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_gcp_cloud_cost.md:49]: Refer to the previous comments.

**Review by rashminandasahoo — CHANGES_REQUESTED** [review]

**Review by rashminandasahoo — APPROVED** [review]

**Pull request merged**